osgeo_importer/templates/osgeo_importer/new.html

```diff
@@ -48,9 +48,6 @@ <h2 class="page-title">{% trans "Add data" %}</h2>
                 <i class="fa fa-warning">  {{ error|escape }}</i>
             {% endfor %}
             </div>
-            <div class="col-md-6" style="padding-top: 5px; padding-left: 2px;">
```

Comment:
@harts-boundless this is resolved at https://github.com/boundlessgeo/django-osgeo-importer/pull/20/commits/39522d2e31d3b80df4664d25d64377e145ac700d but I don't know how to kill this comment thread because new-GitHub is silly